docs/datamodel/links.rst

```diff
@@ -274,8 +274,34 @@ same "shirt owner" relationship is represented with a ``multi`` link.
 Under the hood, a ``multi`` link is stored in an intermediate `association
 table <https://en.wikipedia.org/wiki/Associative_entity>`_, whereas a
 ``single`` link is stored as a column in the object type where it is declared.
-As a result, single links are marginally more efficient. Generally ``single``
-links are recommended when modeling 1:N relations.
+
+.. note::
+
+  Choosing a link direction can be tricky when dealing with these kinds of
+  links. Should you model the link as a one-to-many using ``multi``, or should
+  you model a many-to-one (single) link and use backlinks to traverse in the
+  other direction? A general rule of thumb in this case is as follows.
```

Comment:
A few small suggestions for clarity. Mostly trying to tease out how _relationships_ are modeled _using_ links. In other words, you don't "model links" you "model relationships (one-to-many, or many-to-one) with links (multi or single)".

```suggestion
  Choosing a link direction can be tricky when modeling with these kinds of
  relationships. Should you model the relationship as one-to-many using a ``multi`` link, or as
  many-to-one using a ``single`` link with a backlink to traverse in the
  other direction? A general rule of thumb in this case is as follows.
```

—Note that I didn't reformat the line lengths to highlight the changes here, so this suggestion isn't something that can be applied directly.

Reply:
@Dhghomon did you see this suggestion? any thoughts?

Reply:
Weird, was this hidden? I'll commit it now.

Reply:
@Dhghomon Not sure! But, as I noted, the formatting is not quite right, I just edited it inline to highlight the suggested changes, so you'll need to format the line lengths (looking forward to not caring about this soon!)